ansible/group_vars/omero-hosts.yml

```diff
@@ -174,11 +175,13 @@ omero_web_config_set:
 # Plugins and additional web configuration
 
 omero_web_apps_packages:
-- "omero-mapr==0.2.3"
+- "omero-mapr==0.3.0a2"
 - "omero-iviewer==0.6.0"
+- "git+git://github.com/snoopycrimecop/omero-gallery.git@master_merge_daily#egg=omero-gallery"
```

Comment:
```suggestion
- omero-gallery==3.2.0a3
```

Reply:
I'm now manually installing via ```pip install git+git://github.com/snoopycrimecop/omero-gallery.git@master_merge_daily#egg=omero-gallery``` so I guess that's fine by me.